Seasons Defense/Assets/Scripts/Missile.cs

```diff
@@ -10,6 +10,7 @@ public class Missile : MonoBehaviour
     private void Start()
     {
         transform.LookAt(target);
+        speed = 5;
     }
```

Comment:
I probably said this off earlier, but you can just set the speed outside of Start() and init it in the instance variable declarations.
```suggestion
    public int speed = 5;
    
    private void Start()
    {
        transform.LookAt(target);
    }
```